lib/public_key/src/pubkey_cert.erl

```diff
@@ -1805,41 +1805,48 @@ compatible_ext_key_usage(undefined, _, endentity) -> %% keyusage (first arg )is
     true;
```

Comment:
line above
`compatible_ext_key_usage(undefined, _, endentity) -> %% keyusage (first arg )is mandantory in CAs`

- is confusing. what is the intention? is it after accepting end entity cert with no KeyUsage extension?
- why code comment refers to CAs if function head matches endentity?
- typo: mandantory -> mandatory

Reply:
I think the comment is from before the last argument was added, I can remove it.